clusterloader2/testing/load/config.yaml

```diff
@@ -61,14 +60,9 @@
 # TODO( https://github.com/kubernetes/perf-tests/issues/1027): Lower the number of "min-pods" once we fix the scheduler throughput measurement.
 {{$totalSchedulerThroughputPods := MaxInt (MultiplyInt 2 $MIN_PODS_IN_SMALL_CLUSTERS) .Nodes}}
 {{$schedulerThroughputReplicasPerNamespace := 1}}
-{{$schedulerThroughputPodsPerDeployment := .Nodes}}
+{{$schedulerThroughputPodsPerDeployment := MinInt $totalSchedulerThroughputPods $SCHEDULER_THROUGHPUT_PODS_PER_DEPLOYMENT}}
```

Comment:
If `.Nodes < 1000` then `min(max(1000, .Nodes), .Nodes)` is `.Nodes`. If I understood PR description well, we should have 1,000 pods for clusters with less than 1,000 nodes. So we should probably have `MaxInt` here?

Reply:
Fixed.